Update login form fields to match core components

adamwoodnz opened the pull request:
<!--
Link a related Github/Linear issue to this PR. If the PR does not immediately resolve the issue,
for example, it requires a separate deployment to production, avoid
using the "fixes" keyword. Consider using "part of" instead.
-->

Fixes [DSGCOM-95: Update text fields on login](https://linear.app/a8c/issue/DSGCOM-95/update-text-fields-on-login) and [DOTCOM-13472: Align Blaze Pro input field states with default login](https://linear.app/a8c/issue/DOTCOM-13472/align-blaze-pro-input-field-states-with-default-login)

Alternative to https://github.com/Automattic/wp-calypso/pull/103319, using core styles only and not replacing the field components, as [the core components aren't compatible with Server Side Rendering](https://linear.app/a8c/issue/DSGCOM-95/update-text-fields-on-login#comment-946aa4c2).

Part of https://linear.app/a8c/issue/DOTCOM-13221/log-in-audit-and-cleanup

## Proposed Changes

* Adds an opt in prop to `FormTextInput` and `FormLabel` which if passed styles them using `@wordpress/base-styles`
* Updates branding where applicable
* Makes the password visibility toggle keyboard accessible
* Adds a visible focus style to the 'Change username/email' button

Recording made using keyboard only to move between buttons and inputs:

https://github.com/user-attachments/assets/0af1d7a6-bca2-4e30-b9d9-4851f9ff4da7

## Why are these changes being made?
<!--
It's easy to see what a PR does but much harder to find out why it was made,
particularly when researching old changes in history. Record an explanation of
the motivation behind this change and how it will help.
-->

Part of https://linear.app/a8c/issue/DOTCOM-13221/log-in-audit-and-cleanup

* The current focus styles of the fields lack sufficient contrast, and aren't consistent with the core buttons now used on all login screens
* The password visibility toggle is not accessible

## Testing

<!--
Add as many details as possible to help others reproduce the issue and test the fix.
"Before / After" screenshots can also be very helpful when the change is visual.
-->

### Screenshots

| Default | Inputs and buttons focused | With username |
|-|-|-|
| ![calypso localhost_3000_log-in_redirect_to=%2Fsetup%2Fonboarding%2Fdomains signup_url=%2Fsetup%2Fonboarding%2Fuser(Desktop) (2)](https://github.com/user-attachments/assets/f7db63c2-fe77-4d60-91c2-fe7801100177) | ![calypso localhost_3000_log-in_redirect_to=%2Fsetup%2Fonboarding%2Fdomains signup_url=%2Fsetup%2Fonboarding%2Fuser(Desktop) (3)](https://github.com/user-attachments/assets/ad618487-2e2e-4576-aa5a-32655a3b051f) | ![calypso localhost_3000_log-in(Desktop) (9)](https://github.com/user-attachments/assets/39606bcc-c5df-4776-b471-373176269a1b) |
| ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D1c22dc5ce0975240c46902a714d400aa844e01ee0e530cfbfd8a6890603fa7cc%26redirect_uri%3D](https://github.com/user-attachments/assets/afd352f4-5e9d-4507-a237-4889c206fb04) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D1c22dc5ce0975240c46902a714d400aa844e01ee0e530cfbfd8a6890603fa7cc%26redirect_ur (1)](https://github.com/user-attachments/assets/8821e73c-ac34-4bdf-a76a-0bfd7676e252) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D1c22dc5ce0975240c46902a714d400aa844e01ee0e530cfbfd8a6890603fa7cc%26redirect_ur (3)](https://github.com/user-attachments/assets/a6924e6c-75cd-4718-9729-ecc2f51034e7) |
| ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26client_id%3D99370%26state%3D6e545c7a9932a0a612817faa07992930afae8229e4b11a219e9db95944038cb9%26redirect_uri%3 (7)](https://github.com/user-attachments/assets/f75d158c-d728-4154-b986-824162ab406b) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26client_id%3D99370%26state%3D6e545c7a9932a0a612817faa07992930afae8229e4b11a219e9db95944038cb9%26redirect_uri%3 (8)](https://github.com/user-attachments/assets/a6641f65-0e41-4c60-8f3f-c753b56f5af7) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26client_id%3D99370%26state%3D6e545c7a9932a0a612817faa07992930afae8229e4b11a219e9db95944038cb9%26redirect_uri% (10)](https://github.com/user-attachments/assets/ecd8e6aa-6e7c-4aa4-b0da-af3d4000aa1a) |
| ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fr-login wordpress com%2Fremote-login php%3Faction%3Dlink%26back%3Dhttps%253A%252F%252Fakismet com%252Faccount%252F signup_url=%2Fstart%2Faccount%2Fuser-social%3Fredirect_to%3Dhttps (4)](https://github.com/user-attachments/assets/5bdd2c57-3301-47a9-a26d-c133cc3bb1d1) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fr-login wordpress com%2Fremote-login php%3Faction%3Dlink%26back%3Dhttps%253A%252F%252Fakismet com%252Faccount%252F signup_url=%2Fstart%2Faccount%2Fuser-social%3Fredirect_to%3Dhttps (5)](https://github.com/user-attachments/assets/9943c204-2386-46f4-a0c4-a2ccca38b10c) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fr-login wordpress com%2Fremote-login php%3Faction%3Dlink%26back%3Dhttps%253A%252F%252Fakismet com%252Faccount%252F signup_url=%2Fstart%2Faccount%2Fuser-social%3Fredirect_to%3Dhttps (7)](https://github.com/user-attachments/assets/08c1a1f5-b405-46bf-b8e1-f30726743e3b) |
| ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D95931%26redirect_uri%3Dhttps%253A%252F%252Fagencies automattic com%252Fconnect%252Foauth%252Ftok (2)](https://github.com/user-attachments/assets/e9d7828b-f665-48d5-985e-627cceb42e01) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D95931%26redirect_uri%3Dhttps%253A%252F%252Fagencies automattic com%252Fconnect%252Foauth%252Ftok (3)](https://github.com/user-attachments/assets/dc93ea50-e83e-4ab4-8085-36feba6b320e) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D95931%26redirect_uri%3Dhttps%253A%252F%252Fagencies automattic com%252Fconnect%252Foauth%252Ftok (4)](https://github.com/user-attachments/assets/a8cdd4ed-4e87-4d49-a238-b54b7d932035) |
| ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud jetpack com%252Fconnect%252Foauth%252Ftoken%253 (1)](https://github.com/user-attachments/assets/24e864dc-0949-49c3-8e1a-d7cde12babee) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud jetpack com%252Fconnect%252Foauth%252Ftoken%253 (2)](https://github.com/user-attachments/assets/77966895-e440-4eaf-9615-f59f95cd6b6e) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud jetpack com%252Fconnect%252Foauth%252Ftoken%253 (3)](https://github.com/user-attachments/assets/88be7725-1fc4-480c-8fb7-f39437f89fca) |
| ![calypso localhost_3000_log-in_jetpack(Desktop) (1)](https://github.com/user-attachments/assets/81c0e5b7-58f3-469b-b927-f186c7fa69a0) | ![calypso localhost_3000_log-in_jetpack(Desktop) (2)](https://github.com/user-attachments/assets/ea3717c8-a381-4438-ad3f-cb369e8e87a1) | ![calypso localhost_3000_log-in_link_username_only=true(Desktop)](https://github.com/user-attachments/assets/12c23dd6-1900-4b66-bffd-89348a620e05) |
| ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fclient_id%3D1854%26response_type%3Dcode%26blog_id%3D0%26state%3D480d2703f1dddd949a6088e43a1e68bc311e4502112c5576a820476e043fd16c%26redir](https://github.com/user-attachments/assets/fae5199e-093e-4377-9bf6-a5f7ea116168) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fclient_id%3D1854%26response_type%3Dcode%26blog_id%3D0%26state%3D480d2703f1dddd949a6088e43a1e68bc311e4502112c5576a820476e043fd16c%26r (1)](https://github.com/user-attachments/assets/74035caf-e610-4114-8677-4a497057f305) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fclient_id%3D1854%26response_type%3Dcode%26blog_id%3D0%26state%3D480d2703f1dddd949a6088e43a1e68bc311e4502112c5576a820476e043fd16c%26r (2)](https://github.com/user-attachments/assets/60905bd3-3ab7-469f-a266-5f76bc777b32) |
| ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%2F%3Fclient_id%3D90057%26response_type%3Dcode%26state%3DeyJyZWRpcmVjdF90byI6Imh0dHBzOlwvXC93cGpvYm1hbmFnZXIuY29tXC9teS1hY2NvdW50In0%26 (6)](https://github.com/user-attachments/assets/b9161b74-3d2d-4d81-8a8c-9e7386f8a3fd) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%2F%3Fclient_id%3D90057%26response_type%3Dcode%26state%3DeyJyZWRpcmVjdF90byI6Imh0dHBzOlwvXC93cGpvYm1hbmFnZXIuY29tXC9teS1hY2NvdW50In0%26 (7)](https://github.com/user-attachments/assets/5c8f5a16-d1d4-4ab3-a5cc-bec0d24d8109) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%2F%3Fclient_id%3D90057%26response_type%3Dcode%26state%3DeyJyZWRpcmVjdF90byI6Imh0dHBzOlwvXC93cGpvYm1hbmFnZXIuY29tXC9teS1hY2NvdW50In0%26 (8)](https://github.com/user-attachments/assets/ab49175a-ce54-48ff-aefa-77a43ca55831) |
| ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fclient_id%3D978%26response_type%3Dcode%26blog_id%3D0%26state%3D4d83e8088443caf5ec4ff3581e3973b1c5c896dc4d1201fdf95655735daba91e%26re (2)](https://github.com/user-attachments/assets/2747b3fe-1a48-436b-b6b0-a9fc5fcbc40e) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fclient_id%3D978%26response_type%3Dcode%26blog_id%3D0%26state%3D4d83e8088443caf5ec4ff3581e3973b1c5c896dc4d1201fdf95655735daba91e%26re (3)](https://github.com/user-attachments/assets/98b8260c-953e-4c3f-a52c-b7bf32de169f) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fclient_id%3D978%26response_type%3Dcode%26blog_id%3D0%26state%3D4d83e8088443caf5ec4ff3581e3973b1c5c896dc4d1201fdf95655735daba91e%26re (4)](https://github.com/user-attachments/assets/6d64af5a-ff55-44bc-9abf-523e1eea99a6) |

### Instructions

1. Click each of the login links below
2. Interact with the text fields:

- check the focus state matches the primary button and social buttons
- submit username unfilled to check the error state
- enter text to clear the error state
- submit the username to check the disabled state
- submit the form again to check the password error state
- enter a password to clear the error state, and use the visibility toggle to view the password, using both mouse and keyboard
- submit again and view the disabled state of the input fields while the request is in progress

3. Other checks

- Tab through the form when it first loads and check that the password visibility toggle is not in the tabindex
- Activate Voiceover/JAWS/NVDA and run through step 2 again listening to the interactions
- Check that the 'Change username/email' button (displayed when a username or email has been entered) receives focus using the keyboard and has visible focus styles

#### Login links

[WordPress.com](http://calypso.localhost:3000/log-in)
[Crowdsignal](http://calypso.localhost:3000/log-in?client_id=978&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D978%26response_type%3Dcode%26blog_id%3D0%26state%3Df5da55616562ed1b1ef36c8e88328853efcc0167150831e59437c3330ec11509%26redirect_uri%3Dhttps%253A%252F%252Fapp.crowdsignal.com%252Fconnect%253Fsource%253Dlogin%2526action%253Drequest_access_token%26wpcom_connect%3D1%26from-calypso%3D1)
[Jetpack Cloud](http://calypso.localhost:3000/log-in?redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud.jetpack.com%252Fconnect%252Foauth%252Ftoken%253Fnext%253D%25252F%26scope%3Dglobal%26blog_id%3D0%26from-calypso%3D1&client_id=69040&signup_url=%2Fstart%2Fwpcc%2Foauth2-user%3Foauth2_client_id%3D69040%26oauth2_redirect%3Dhttps%253A%252F%252Fpublic-api.wordpress.com%252Foauth2%252Fauthorize%253Fresponse_type%253Dtoken%2526client_id%253D69040%2526redirect_uri%253Dhttps%25253A%25252F%25252Fcloud.jetpack.com%25252Fconnect%25252Foauth%25252Ftoken%25253Fnext%25253D%2525252F%2526scope%253Dglobal%2526blog_id%253D0%2526from-calypso%253D1)
[Woo](http://calypso.localhost:3000/log-in?redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3Db30d4e6a81f72b803c873f517f4189601c46a1bc8b9bfdb9ed27cf3d17ad5601%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253D%25252F%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26from-calypso%3D1&client_id=50916&signup_url=%2Fstart%2Fwpcc%2Foauth2-user%3Foauth2_client_id%3D50916%26oauth2_redirect%3Dhttps%253A%252F%252Fpublic-api.wordpress.com%252Foauth2%252Fauthorize%252F%253Fresponse_type%253Dcode%2526client_id%253D50916%2526state%253Db30d4e6a81f72b803c873f517f4189601c46a1bc8b9bfdb9ed27cf3d17ad5601%2526redirect_uri%253Dhttps%25253A%25252F%25252Fwoocommerce.com%25252Fwc-api%25252Fwpcom-signin%25253Fnext%25253D%2525252F%2526blog_id%253D0%2526wpcom_connect%253D1%2526wccom-from%2526calypso_env%253Dproduction%2526from-calypso%253D1)
[Blaze Pro](http://calypso.localhost:3000/log-in?redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26client_id%3D99370%26state%3D0f0d1f16975b65cf8e9e924461b7c892f78eabeeb190dc03654cef0465786da0%26redirect_uri%3Dhttps%253A%252F%252Fblazepro.tumblr.com%252Fauth%252Fconnected%26blog_id%3D0%26wpcom_connect%3D1%26calypso_env%3Dproduction%26scope%3Dglobal%26from-calypso%3D1&client_id=99370&signup_url=%2Fstart%2Fwpcc%2Foauth2-user%3Foauth2_client_id%3D99370%26oauth2_redirect%3Dhttps%253A%252F%252Fpublic-api.wordpress.com%252Foauth2%252Fauthorize%253Fresponse_type%253Dcode%2526client_id%253D99370%2526state%253D0f0d1f16975b65cf8e9e924461b7c892f78eabeeb190dc03654cef0465786da0%2526redirect_uri%253Dhttps%25253A%25252F%25252Fblazepro.tumblr.com%25252Fauth%25252Fconnected%2526blog_id%253D0%2526wpcom_connect%253D1%2526calypso_env%253Dproduction%2526scope%253Dglobal%2526from-calypso%253D1)
[Akismet](http://calypso.localhost:3000/log-in?redirect_to=https%3A%2F%2Fr-login.wordpress.com%2Fremote-login.php%3Faction%3Dlink%26back%3Dhttps%253A%252F%252Fakismet.com%252Faccount%252F&signup_url=%2Fstart%2Faccount%2Fuser-social%3Fredirect_to%3Dhttps%253A%252F%252Fr-login.wordpress.com%252Fremote-login.php%253Faction%253Dlink%2526back%253Dhttps%25253A%25252F%25252Fakismet.com%25252Faccount%25252F)
A4A - Can't link due to GitHub OSS scan rules: go to automatt_c.com/for-agencies/ and click 'Log in'. Edit the URL and change the host to calypso.localhost:3000
[Gravatar](http://calypso.localhost:3000/log-in?redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D1854%26response_type%3Dcode%26blog_id%3D0%26state%3D4a5153e30a28e2ef24ca7e599abd80f3969340cd1f88327c1f5d2fbc88593fa1%26redirect_uri%3Dhttps%253A%252F%252Fgravatar.com%252Fconnect%252F%253Faction%253Drequest_access_token%26from-calypso%3D1&client_id=1854)
[WP Job Manager](http://calypso.localhost:3000/log-in?redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fclient_id%3D90057%26response_type%3Dcode%26state%3DeyJyZWRpcmVjdF90byI6Imh0dHBzOlwvXC93cGpvYm1hbmFnZXIuY29tXC9teS1hY2NvdW50In0%26redirect_uri%3Dhttps%253A%252F%252Fwpjobmanager.com%252Fwp-json%252Fwpjmcom-auth%252Fv1%252Fcallback%26blog_id%3D0%26from-calypso%3D1&client_id=90057)
[VIP](http://calypso.localhost:3000/log-in?redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fstate%3Dd01a004590fc4017316cef309b1e213de0cf6c1fd4b68d3152d9f9acbf9c%26scope%3Dauth%26response_type%3Dcode%26approval_prompt%3Dauto%26redirect_uri%3Dhttps%253A%252F%252Fid.wpvip.com%252Fwp-json%252Foauth%252Fv1%252Fcallback%252Fwpcom%26client_id%3D76596%26from-calypso%3D1&client_id=76596&signup_url=%2Fstart%2Fwpcc%2Foauth2-user%3Foauth2_client_id%3D76596%26oauth2_redirect%3Dhttps%253A%252F%252Fpublic-api.wordpress.com%252Foauth2%252Fauthorize%253Fstate%253Dd01a004590fc4017316cef309b1e213de0cf6c1fd4b68d3152d9f9acbf9c%2526scope%253Dauth%2526response_type%253Dcode%2526approval_prompt%253Dauto%2526redirect_uri%253Dhttps%25253A%25252F%25252Fid.wpvip.com%25252Fwp-json%25252Foauth%25252Fv1%25252Fcallback%25252Fwpcom%2526client_id%253D76596%2526from-calypso%253D1)
Partner Portal - Can't link due to GitHub OSS scan rules: go to hosts.automatt_c.com and you should be redirected to log in. Edit the URL and change the host to calypso.localhost:3000
WP.Cloud - Can't link due to GitHub OSS scan rules: go to [wp.cloud](https://wp.cloud) and click 'start here'. Edit the URL and change the host to calypso.localhost:3000

## Pre-merge Checklist

<!--
Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
-->

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
    - [ ] For UI changes, have we tested the change in various languages (for example, ES, PT, FR, or DE)? The length of text and words vary significantly between languages. 
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

adamwoodnz committed:
Add opt in core styles to form text inputs

adamwoodnz commented on client/assets/stylesheets/shared/_extends-forms.scss:
```diff
@@ -82,3 +90,28 @@
+%form-field-core-styles {
+	@include input-control;
+
+	box-sizing: border-box;
+	width: 100%;
+	height: $grid-unit-50;
+	border-color: var(--color-neutral-10);
+	font-size: $font-body-small;
+	color: var(--color-neutral-60);
+
+	@include break-small {
+		font-size: $font-body-small;
+	}
```
All based on the design: YIwc478nGHyjIZ45hQ1hfV-fi-245_3240

ciampo replied:
Not necessarily feedback for this PR, but more of a general comment:

Assuming the Figma specs are using the DS specs (WP or A8C), I assume that, if we migrated to using the components from `@wordpress/components` and `@automattic/components/` packages, there should little-to-no need for these overrides?

adamwoodnz replied:
Correct, these styles make the inputs look essentially exactly the same as they did in this [previous abandoned PR](https://github.com/Automattic/wp-calypso/pull/103319) using the core components.

adamwoodnz commented on client/assets/stylesheets/shared/_extends-forms.scss:
```diff
@@ -82,3 +90,28 @@
+		font-size: $font-body-small;
+	}
+
+	@extend %form-field-disabled;
```
Core styles don't seem to include disabled state, so extending the existing

ciampo replied:
I'm not sure about the exact list of Core components you're referring to, but some components should already ship with `disabled` styles (for example, [`InputControl`](https://wordpress.github.io/gutenberg/?path=/story/components-experimental-inputcontrol--default&args=label:Value;placeholder:Placeholder;__next40pxDefaultSize:!true;disabled:!true))

adamwoodnz replied:
Ah I see now; styles are added to the control container which we don't have here using the existing form controls. I'll see if I can also use a mixin for that.

adamwoodnz replied:
These styles are set in [JS](https://github.com/WordPress/gutenberg/blob/a3b72eeef3b69d524e88f8120cb62a0442bc9f11/packages/components/src/input-control/styles/input-control-styles.tsx#L95), so I've had to copy them in, see [cf76046](https://github.com/Automattic/wp-calypso/pull/103616/commits/cf76046f127673b3d9069f98512020b37b805cde)

adamwoodnz committed:
Add product branding overrides

adamwoodnz commented on client/assets/stylesheets/_wp-base-styles-overrides.scss:
Follows the approach of centralizing overrides from https://github.com/Automattic/wp-calypso/pull/103611

adamwoodnz commented on client/assets/stylesheets/_wp-base-styles-overrides.scss:
Follows the approach of centralizing overrides from https://github.com/Automattic/wp-calypso/pull/103611

adamwoodnz committed:
Fix base-styles import paths

adamwoodnz committed:
Try adding node_modules to reader mobile build

adamwoodnz commented on bin/render-stylesheet.js:
```diff
@@ -37,10 +37,18 @@ const importer = ( url, prev, done ) => {
+		file: args.in,
+		importer,
+		outputStyle: 'compressed',
+		includePaths: [ 'node_modules' ],
```
Team City build for reader mobile stylesheet was failing without this, as it imports `_extend-forms.scss`

jasmussen commented:
Very nice. I appreciate continuing with the singular "style overrides" SCSS file as a singular place to store these. I envision these also eventually being able to update core PHP component styles.

Should we update the label to the unified all-caps style that the core components have?

One thing also, both help text for inputs, but also the validation pattern here, should have `text-wrap: pretty;` applied. Both in the these overrides, but also in the components themselves. CC: @mirka as I recall you working on the validation patterns. 

🙏 Thanks for keeping on this!

jasmussen commented:
Added note related to labels, as part of [content guidelines](https://linear.app/a8c/project/content-guidelines-6dcfdb2ae02a/overview), we should move from titlecase to sentence case wherever we encounter it. Not a blocker at all, but noting, as it would affect at least this one:

* Email Address or Username → Email address or username

These should be sentence case even if we use CSS to all-caps it. Painting the back of the fence here :D

jasmussen commented:
Also curious, @michaelpick, if it should be "Privacy policy" and "Terms of service"? Even as part of a sentence, should it then be "By continuing you agree to our [link]terms of service[/link] ..." ?

adamwoodnz committed:
Update username field label case

adamwoodnz commented:
> Should we update the label to the unified all-caps style that the core components have?

Not sure about this, to my eye it looks very shouty, especially with other brand styles, eg. A4A

### Screenshots

| .com | Woo | A4A | Crowdsignal |
|-|-|-|-|
| ![Screenshot 2025-05-26 at 3 46 25 PM](https://github.com/user-attachments/assets/48d02be4-40e0-4b66-8ba4-91dbbc3ae80b) | ![Screenshot 2025-05-26 at 3 46 34 PM](https://github.com/user-attachments/assets/ab8154b8-f1a7-44bb-ad24-918565d6c53f) | ![Screenshot 2025-05-26 at 3 46 51 PM](https://github.com/user-attachments/assets/9a0251a2-885f-4468-a82e-24fc6e146942) | ![Screenshot 2025-05-26 at 3 47 09 PM](https://github.com/user-attachments/assets/93fb19cd-63e7-48cd-83b6-023334ff6b15) |

The design (YIwc478nGHyjIZ45hQ1hfV-fi-245_1996) shows sentence case, what do you think?

adamwoodnz committed:
Add opt in core styles for label

adamwoodnz committed:
Stop spreading prop to DOM element

adamwoodnz committed:
Add story for label with core styles

adamwoodnz committed:
Theme Woo

adamwoodnz committed:
Theme Akismet

adamwoodnz committed:
Fix spacing on signup form

adamwoodnz committed:
Theme A4A

adamwoodnz committed:
Adjust scope of overrides

adamwoodnz committed:
Use core styles for password label

adamwoodnz committed:
Theme Jetpack Cloud

adamwoodnz committed:
Theme Jetpack login

adamwoodnz committed:
Fix error states

adamwoodnz committed:
Theme Gravatar

jasmussen commented:
> Not sure about this, to my eye it looks shouty, especially with other brand styles, eg. A4A

I don't see it that much as shouty, one of the benefits especially when the labels are shorter is that it becomes more of a bar, optically. Given an incoming unification of these screens that style should still be part of it: p58i-lKQ-p2

That said, thanks for surfacing the inconsistency added a comment in the Figma. The goal with this work is in part getting the details right, but more importantly it's about reducing the unintentional divergence and maintenance that comes with that. Any exception, thus, has a small price tag worth being sure about!

adamwoodnz committed:
Theme WP Job Manager

adamwoodnz committed:
Update change username button

- Use sentence case
- Use core button for a11y

adamwoodnz committed:
Make password visibility toggle accessible

adamwoodnz committed:
Theme Woo password toggle

adamwoodnz committed:
Remove custom colors for change username link

adamwoodnz committed:
Increase separation between password manager and visibility toggle

jasmussen commented:
Followed up with @fcoveram, and also looped in @nuriapenya @fditrapani for general awareness since they've been a bit involved with related screens. The answer to whether sentence case or all-caps is: we should make a decision but then apply that to all these related screens. The DS defaults to all-caps for the `label` element, which is why I'm leaning that way. But it's not a strong opinion, the key is that the CSS file that holds these "logged out DS components" is _the same_ and will apply also to related wp.com connect efforts. Let me know if that all makes sense/helps, and what opinions lean towards 🙏

fcoveram commented:
Seconding Joen's comment above and the following approach

> …whether sentence case or all-caps is: we should make a decision but then apply that to all these related screens…

nuriapenya commented:
> Not sure about this, to my eye it looks shouty, especially with other brand styles, eg. A4A

I agree, but I believe it's because the input field it's alone and kind of isolated. When we look at other forms with more input fields, all-caps work usually fine, **as long as we use short labels**. 

The most important thing, as @jasmussen mentioned, is to make sure we are using the same source of truth and it's apply in all flow/screens.

michaelpick commented:
+1 to consistency of labels @jasmussen  and @fcoveram. Is the use case for the caps just a historical one or an industry trend? From a readability POV, sentence case is generally the better call. 

That said, if there's a reason to keep caps from a design POV, I agree with @nuriapenya that we need to keep them short. The longer ones do feel very shouty to me.

Re:

> Also curious, @michaelpick, if it should be "Privacy policy" and "Terms of service"? Even as part of a sentence, should it then be "By continuing you agree to our [link]terms of service[/link] ..." ?

(Sorry for the late reply, I was out yesterday)

It makes sense to me that we'd include both as requirements, but might be good to get legal and/or Happiness insight on this one if we're not 100%. I think the privacy policy requirement might be an EU directive inherited from GDPR, whereas ToS define broader platform-level terms of use.

jasmussen commented:
To note as a followup, and based on the above emerging consensus related to the appearance of input labels. The default for the DS is to show these as [small allcaps](https://wordpress.github.io/gutenberg/?path=/docs/components-basecontrol--docs), which is useful in most cases. But perhaps we need a variant for when the input stands alone, outside of a base control or a card, like the login, where the label remains sentence case.

What do you think?

jasmussen commented:
CC: @ciampo @mirka for thoughts on whether two types of label styles are reasonable, depending on context.

ciampo commented on client/assets/stylesheets/_wp-base-styles-overrides.scss:
What's the relationship between this file and `client/assets/stylesheets/_wp-components-overrides.scss` ?

I see that we're applying some overrides to all `input` elements, and I wonder if those overrides are also expected to "bleed" into components from the `@wordpress/components` package.

adamwoodnz replied:
The overrides in this file at present do not affect `@wordpress/components`, and only override `--wp-*` vars set by the base styles package.

> I see that we're applying some overrides to all input elements, and I wonder if those overrides are also expected to "bleed" into components from the @wordpress/components package.

They're intentionally not overriding all input elements or bleeding into `@wordpress/components` at present; only those with the opt in class `.form-text-input-core-styles` are affected. For the core input components I imagine we'll want to add separate overrides to `_wp-components-overrides.scss`, but they're not in use in this screen due to the SSR incompatibility.

I think some documentation in this file would help, and perhaps it should be restructured so that the outer selector is `.form-text-input-core-styles`

```
/* Extends @wordpress/base-styles/mixins input-control() */
.form-text-input-core-styles {
	.blaze-pro &,
	.is-blaze-pro & {
		&:not(.is-error) {
			--wp-admin-theme-color: var(--color-gray);
		}
	}

	.woo.is-woo-passwordless & {
		...
	}
}
```

adamwoodnz replied:
[Refactor and docs added](https://github.com/Automattic/wp-calypso/pull/103616/commits/d40ec8f1588b63ddbaa061fef5d2d6ecc61c480c)

ciampo commented on client/assets/stylesheets/_wp-base-styles-overrides.scss:
```diff
@@ -0,0 +1,80 @@
+.woo.is-woo-passwordless {
+	input[type='text'],
+	input[type='password'] {
+		&.form-text-input-core-styles {
+			/* stylelint-disable-next-line scales/radii */
+			border-radius: 8px;
+			font-size: 1rem;
+			height: 48px;
+			padding: 8px 16px;
+
+			&:not(:focus, :disabled) {
+				border-color: var(--studio-gray-30);
+			}
+
+			&:focus {
+				border-width: 1.5px;
+			}
+
+			&:not(.is-error) {
+				--wp-admin-theme-color: var(--woo-purple-50);
+			}
```
Overriding aspects beyond colors, such as border radius, font-size, height, and padding makes me a bit uncomfortable — why are those overrides needed? Can we just use existing components instead, with a light "theming" touch through color overrides?

adamwoodnz replied:
Woo has heavy customisations of both inputs and buttons:

| Default | Focused |
|-|-|
| ![Screenshot 2025-05-29 at 1 22 13 PM](https://github.com/user-attachments/assets/48a41893-245e-4a26-afa1-cff3418a04ad) | ![Screenshot 2025-05-29 at 1 23 49 PM](https://github.com/user-attachments/assets/180ee226-cea5-43b2-b2f8-eb75e017915c) |

I agree it's quite a maintenance overhead, but I'm not sure if this is the PR to change that, as the scope will be large as it is. I think it's helpful to at least have the overrides centralized here, and we can look to reduce them in consultation with Woo design?

ciampo commented on client/assets/stylesheets/_wp-base-styles-overrides.scss:
```diff
@@ -0,0 +1,80 @@
+	input[type='password'] {
+		&.form-text-input-core-styles {
+			&:not(:focus, :disabled) {
+				border-color: #9ea4a8;
```
If there's no variable for this color, maybe we can add an inline comment explaining if there's at least a reference to is in the design specs?

adamwoodnz replied:
I tried to maintain the original, but I think there's a strong case here for using the default.

adamwoodnz replied:
Removed in [4cae5aa](https://github.com/Automattic/wp-calypso/pull/103616/commits/4cae5aa78a2b33c7e1b041b9e222735c5eae3df4)

ciampo commented on client/assets/stylesheets/style.scss:
```diff
@@ -19,3 +19,6 @@
 
 // Overrides for @wordpress/components
 @import "./wp-components-overrides";
+
+// Overrides for @wordpress/base-styles
+@import "./wp-base-styles-overrides";
```
Should the base style overrides be applied before the component overrides?

adamwoodnz replied:
Maybe? I haven't seen any adverse affects, but that's probably because the base overrides are all very targeted and don't affect the components at this point. I'll swap them in case that changes in future.

adamwoodnz replied:
Done in [0fbcae7](https://github.com/Automattic/wp-calypso/pull/103616/commits/0fbcae7111d2dbc424c4e3f4b7eee104b3eb3a98)

ciampo commented on client/blocks/login/login-form.jsx:
```diff
@@ -430,7 +436,7 @@ export class LoginForm extends Component {
 						<FormLabel htmlFor="usernameOrEmail">
 							{ this.isPasswordView() ? (
-								<Button
+								<A8CButton
```
Given that `Button` from `@automattic/components` is deprecated, can this be an opportunity to switch to the `Button` component from `@wordpress/components` ?

adamwoodnz replied:
This is a different Woo form to the one I've seen in production, and as such I haven't touched it at all. I've intentionally tried to keep to a limited scope but I'm aware this has likely created inconsistencies. I'll try to find out how to test this flow.

adamwoodnz replied:
This turns out be [very intensive to test](https://github.com/Automattic/wp-calypso/pull/41798) and I have not managed to do so, but I can force this state locally and the button replacements are straightforward so I've [committed](https://github.com/Automattic/wp-calypso/pull/103616/commits/02d847ecc282597d301ee0d706f634ae31d0a107) them.

ciampo commented on client/assets/stylesheets/shared/_extends-forms.scss:
Not specific feedback for this PR, but I'm generally uncomfortable with the amount of style overrides that we apply to form components.

I'd love it if we could move to re-using existing shared componentry with a minimum amount of style overrides (mostly for theming purposes)

adamwoodnz replied:
I've added more screenshots in the description to demonstrate all the variants we currently have for the brands.

ciampo commented:
> CC: @ciampo @mirka for thoughts on whether two types of label styles are reasonable, depending on context.

Quoting this reply that I left earlier on p1748419987948939/1748419029.140069-slack-C05QAFZSFTL :

> We've been talking about writing a new version of BaseControl for some time, making it more modular, more flexible around supported layouts, and capable of labelling "composed" inputs: https://github.com/WordPress/gutenberg/issues/60836
>
> As many other DS and components related things, it's a matter of prioritization. We also recently [discussed](https://github.com/Automattic/wp-calypso/pull/100771#issuecomment-2909185842) making tweaks to existing input controls, so maybe the changes to BaseControl could be part of that body of work.

adamwoodnz committed:
Remove custom Gravatar input border color

adamwoodnz committed:
Refactor base styles overrides and add docs

adamwoodnz committed:
Revert extending existing disabled styles

Copy styles from core instead

adamwoodnz committed:
Add valid styles to maintain parity with existing

adamwoodnz committed:
Reverse order of core override imports

adamwoodnz commented:
> To note as a followup, and based on the above emerging consensus related to the appearance of input labels. The default for the DS is to show these as [small allcaps](https://wordpress.github.io/gutenberg/?path=/docs/components-basecontrol--docs), which is useful in most cases. But perhaps we need a variant for when the input stands alone, outside of a base control or a card, like the login, where the label remains sentence case.
> 
> What do you think?

If we choose not to create a variation and leave the labels small allcaps, I think the 2 variations that really need attention are A4A and Jetpack Cloud which have a larger font size and lighter weight applied:

| A4A | Jetpack Cloud |
|-|-|
| ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D95931%26redirect_uri%3Dhttps%253A%252F%252Fagencies automattic com%252Fconnect%252Foauth%252Ftok (1)](https://github.com/user-attachments/assets/03a0313a-8b1f-4e2c-99a9-97c58dde46f1) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud jetpack com%252Fconnect%252Foauth%252Ftoken%253 (1)](https://github.com/user-attachments/assets/47f872a4-5099-4190-8756-c97b10e5178b) |

I think we should remove these customisations.

adamwoodnz commented on client/components/forms/form-password-input/style.scss:
```diff
@@ -1,33 +1,24 @@
+	.form-text-input[type] {
 		/*!rtl:ignore*/
-		padding-right: 32px;
+		padding-right: 36px;
```
Allows more room for password manager UI

<img src="https://github.com/user-attachments/assets/51cf90c7-fbc0-4fba-9bc6-026622d4c72b" width="500">

adamwoodnz committed:
Revert removing Blaze Pro input focus styles on other screens

jasmussen commented:
> I think we should remove these customisations.

I would agree with that, and this is being discussed also in context of p58i-lKQ-p2. For awareness, @keoshi, though.

adamwoodnz committed:
Restore order of props

adamwoodnz commented on client/components/forms/form-password-input/index.jsx:
```diff
@@ -33,27 +34,33 @@ class FormPasswordInput extends Component {
+						'form-password-input__toggle-visibility': ! hideToggle,
+					} ) }
+					onClick={ this.togglePasswordVisibility }
+					aria-hidden={ isHidden }
+					tabIndex={ isHidden ? -1 : undefined }
```
Ensures the button is not accessible when this field is in the DOM but hidden visually

adamwoodnz committed:
Stop outputting locale attr on password input

adamwoodnz committed:
Update Jetpack login snapshots

adamwoodnz committed:
Fix login form test

Reduce label selector scope to not match 'Show password', and thereby return multiple items

adamwoodnz committed:
Fix email form test

Increase button count as the password toggle is now a button
Select the submit button by text as there are multiple buttons

adamwoodnz commented on client/blocks/login/test/login-form.jsx:
```diff
@@ -17,7 +17,7 @@ describe( 'LoginForm', () => {
 		expect( username ).toBeInTheDocument();
 
-		const password = screen.getByLabelText( /password/i );
+		const password = screen.getByLabelText( /^password$/i );
```
Restricts the match to only 'Password' and not 'Show password'

adamwoodnz committed:
Remove custom styles from Jetpack and A4A labels

adamwoodnz commented:
> > I think we should remove these customisations.
> 
> I would agree with that, and this is being discussed also in context of p58i-lKQ-p2. For awareness, @keoshi, though.

Done in [208eb43](https://github.com/Automattic/wp-calypso/pull/103616/commits/208eb43776483dcfb9260cf5a93ade9faae7c0a1)

If we're not going with small allcaps for these, and we need a variation. Maybe setting a prop on the label component `hasCoreStylesNoCaps` instead of the standard `hasCoreStyles`

adamwoodnz committed:
Use Core Button for Jetpack Woo DNA flow

adamwoodnz committed:
Add Core style variation for no caps

adamwoodnz commented:
It seems the consensus is for the label to not be uppercase in instances like this, so I've added the `hasCoreStylesNoCaps` variation prop to the FormLabel component in [af04fbb](https://github.com/Automattic/wp-calypso/pull/103616/commits/af04fbb0563c2a6575b67f2318bdea7b3d24659c)

<img src="https://github.com/user-attachments/assets/0484ee74-cf48-47c3-9f89-db0588188751" width="500">

I think that's all the feedback addressed. Probably need more testing, especially for the a11y aspects of the password toggle, otherwise I'm hoping this is ready to land.

adamwoodnz commented on packages/components/src/forms/form-label/index.stories.jsx:
```diff
@@ -17,3 +17,15 @@ Default.args = {
 	required: false,
 };
+
+export const WithCoreStyles = Template.bind( {} );
+WithCoreStyles.args = {
+	className: 'button-label',
+	hasCoreStyles: true,
+};
+
+export const WithCoreStylesNoCaps = Template.bind( {} );
+WithCoreStylesNoCaps.args = {
+	className: 'button-label',
+	hasCoreStylesNoCaps: true,
+};
```
![localhost_56833__path=_story_unaudited-form-label--with-core-styles-no-caps(Desktop)](https://github.com/user-attachments/assets/e56a2cb6-55bd-403d-b087-b402cfad4a91)

jasmussen reviewed (approved):
Tests well for me. Big improvement. Thank you.

![test](https://github.com/user-attachments/assets/a51ae647-0e46-4c1c-a7a9-01685da8a76b)

> It seems the consensus is for the label to not be uppercase in instances like this, so I've added the hasCoreStylesNoCaps variation prop to the FormLabel component in [af04fbb](https://github.com/Automattic/wp-calypso/pull/103616/commits/af04fbb0563c2a6575b67f2318bdea7b3d24659c)

Sounds fine to me, though if @mirka or @ciampo has a chance to look I'd appreciate that, just to know this is the right path forward.

adamwoodnz committed:
Remove A4A theming as it does not meet contrast requirements

Button and link overrides were removed in https://github.com/Automattic/wp-calypso/pull/103944

adamwoodnz commented:
Noting that a number of these screens have now moved to unified login, but the changes are still effective:

| Gravatar | WP Job Manager | Crowdsignal | Blaze Pro |
|-|-|-|-|
| ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fclient_id%3D1854%26response_type%3Dcode%26blog_id%3D0%26state%3Dca233e12c9491c52480383d1b5ee1203159bc8facaff9f9c8b4eb00a3537bb2a%26redir](https://github.com/user-attachments/assets/9e62bd7f-c252-4650-be06-6536576763c2) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%2F%3Fclient_id%3D90057%26response_type%3Dcode%26state%3DeyJyZWRpcmVjdF90byI6Imh0dHBzOlwvXC93cGpvYm1hbmFnZXIuY29tXC9teS1hY2NvdW50In0%26 (9)](https://github.com/user-attachments/assets/be25e5c1-5ae3-4bd9-9006-c3270b917663) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fclient_id%3D978%26response_type%3Dcode%26blog_id%3D0%26state%3D4d83e8088443caf5ec4ff3581e3973b1c5c896dc4d1201fdf95655735daba91e%26re (8)](https://github.com/user-attachments/assets/5e9fb93f-e8d1-4520-8ec5-f11999b33262) | ![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26client_id%3D99370%26state%3D6e545c7a9932a0a612817faa07992930afae8229e4b11a219e9db95944038cb9%26redirect_uri% (15)](https://github.com/user-attachments/assets/7d3a41f1-be49-41c2-9df7-e216d99bf2eb) |

ciampo commented on client/blocks/login/login-form.jsx:
```diff
@@ -418,7 +418,7 @@ export class LoginForm extends Component {
 							{ this.isPasswordView() ? (
 								<Button
-									borderless
+									variant="link"
```
For the `Button` component from `@wordpress/components`, we recommend always using the `__next40pxDefaultSize` prop when using the default size, and otherwise using the `size` prop to pick another size ([see Storybook](https://wordpress.github.io/gutenberg/?path=/docs/components-button--docs))

adamwoodnz replied:
Done in [2d06630](https://github.com/Automattic/wp-calypso/pull/103616/commits/2d0663080a97f720b49c6fa68ddaa00e725b6958)

ciampo reviewed (commented):
Code-wise, things are mostly looking good. I'd maybe look into adding the `__next40pxDefaultSize` prop (or a `size` different than `default`), but not much else to add from looking at the code.

In the long term, I hope we can get rid of the majority of the styles and just use the right components instead.

adamwoodnz commented:
> Does this also affect other screens, like "lost your password" or "create your account" or "continue as user"?

No, there are a number of screens where we'll need to add these props to update the styles. I think I have issues for most screens in [Audit and refactor Connect & Log-In to use core components.](https://linear.app/a8c/project/audit-and-refactor-connect-and-log-in-to-use-core-components-3a205c9f502a), but might have missed some. It should be straightforward after this lands, but I've intentionally split this work up by screen to date, to limited the testing scope with so many brand variants.

adamwoodnz committed:
Use compact size for change username buttons

adamwoodnz committed:
Adjust Gravatar styles

adamwoodnz committed:
Fix branded valid states

adamwoodnz commented:
Thanks for reporting everyone. I'll reduce the specificity of that selector to hopefully fix these overrides.

yashwin commented:
> Fixes for form inputs and labels have been deployed.

Thank you!
